goal_src/jak2/engine/ui/bigmap-h.gc

```diff
@@ -199,57 +208,18 @@
-                           :nreg #x5
-                           )
-               )
-          )
-    (set! (-> gp-0 sprite-tmpl gif1)
-          (the uint64 (new 'static 'gif-tag-regs
-                           :regs0 (gif-reg-id rgbaq)
-                           :regs1 (gif-reg-id uv)
-                           :regs2 (gif-reg-id xyz2)
-                           :regs3 (gif-reg-id uv)
-                           :regs4 (gif-reg-id xyz2)
-                           )
-               )
-          )
+    (set! (-> gp-0 sprite-tmpl gif0) (the-as uint #x50ab400000008001))
+    (set! (-> gp-0 sprite-tmpl gif1) (the-as uint #x53531))
     (set! (-> gp-0 draw-tmpl dma-vif dma) (new 'static 'dma-tag :qwc #xa :id (dma-tag-id cnt)))
     (set! (-> gp-0 draw-tmpl dma-vif vif0) (new 'static 'vif-tag))
     (set! (-> gp-0 draw-tmpl dma-vif vif1) (new 'static 'vif-tag :imm #xa :cmd (vif-cmd direct) :msk #x1))
-    (set! (-> gp-0 draw-tmpl gif0)
-          (the uint64 (new 'static 'gif-tag64
-                           :nloop #x1
-                           :eop #x1
-                           :pre #x1
-                           :prim (new 'static 'gs-prim :prim (gs-prim-type tri-strip) :tme #x1 :abe #x1 :fst #x1)
-                           :nreg #x9
-                           )
-               )
-          )
-    (set! (-> gp-0 draw-tmpl gif1) (the uint64 (new 'static 'gif-tag-regs
-                                                    :regs0 (gif-reg-id rgbaq)
-                                                    :regs1 (gif-reg-id uv)
-                                                    :regs2 (gif-reg-id xyz2)
-                                                    :regs3 (gif-reg-id uv)
-                                                    :regs4 (gif-reg-id xyz2)
-                                                    :regs5 (gif-reg-id uv)
-                                                    :regs6 (gif-reg-id xyz2)
-                                                    :regs7 (gif-reg-id uv)
-                                                    :regs8 (gif-reg-id xyz2)
-                                                    )
-                                        )
-          )
+    (set! (-> gp-0 draw-tmpl gif0) (the-as uint #x90aa400000008001))
+    (set! (-> gp-0 draw-tmpl gif1) (the-as uint #x535353531))
     (set! (-> gp-0 adgif-tmpl dma-vif dma) (new 'static 'dma-tag :qwc #x6 :id (dma-tag-id cnt)))
     (set! (-> gp-0 adgif-tmpl dma-vif vif0) (new 'static 'vif-tag))
     (set! (-> gp-0 adgif-tmpl dma-vif vif1) (new 'static 'vif-tag :imm #x6 :cmd (vif-cmd direct) :msk #x1))
-    (set! (-> gp-0 adgif-tmpl gif0) (the uint64 (new 'static 'gif-tag64 :nloop #x5 :eop #x1 :nreg #x1)))
-    (set! (-> gp-0 adgif-tmpl gif1) (the uint64 (new 'static 'gif-tag-regs :regs0 (gif-reg-id a+d))))
+    (set! (-> gp-0 adgif-tmpl gif0) (the-as uint #x1000000000008005))
+    (set! (-> gp-0 adgif-tmpl gif1) (the-as uint 14))
```

Comment:
might be worth keeping the old ones - they are a lot nicer to read.
I've been meaning to add a hack to the decompiler so we don't need to manually do this though...